### DISCUSSION TITLE
[해결] enum은 제페토스크립트에서 어떻게 사용하나요?

### DISCUSSION THREAD

**Discussion by mayko1234** [naverz/zepeto-studio-kor#300] [5. Q&A]
안녕하세요,
enum은 제페토스크립트에서 어떻게 사용하는지 알고싶습니다.
타입스크립트에서 하는 식으로 하니까 컴파일 에러가 나네요. 
![image](https://user-images.githubusercontent.com/97988822/151686797-753243be-db42-4c2c-8bb5-c828a033bfb6.png)
![image](https://user-images.githubusercontent.com/97988822/151686654-c0ce9bcd-c51b-4b04-9ae5-0367e9e5e5c9.png)

**Comment by mayko1234** [comment]
아 찾았습니다! 클래스 내부에서 선언해서 안되는거였어요. 
enum은 클래스 외부에...😊
질문이 삭제가 안되네요... 😅

**Reply by x4cf** [reply]
mayko1234 님의 글이 같은 문제를 겪는 누군가에게 도움이 될거에요 :)
